com.avaloq.tools.ddk.xtext.generator/META-INF/MANIFEST.MF

```diff
@@ -22,7 +22,8 @@ Require-Bundle: org.eclipse.jface,
  com.avaloq.tools.ddk.xtext.ui,
  org.eclipse.xtend.lib,
  org.eclipse.jdt.core,
- org.eclipse.compare
+ org.eclipse.compare,
+ org.apache.commons.lang
```

Comment:
it looks like we can do quite well without the dependency on org.apacher.commons.lang, so I would not add it